dpnp/dpnp_algo/dpnp_algo.pxd

```diff
@@ -45,7 +45,6 @@ cdef extern from "dpnp_iface_fptr.hpp" namespace "DPNPFuncName":  # need this na
         DPNP_FN_ARGSORT
         DPNP_FN_ARGSORT_EXT
         DPNP_FN_ASTYPE
-        DPNP_FN_ASTYPE_EXT
```

Comment:
Do we need to keep mapping in `dpnp/backend/kernels/dpnp_krnl_elemwise.cpp` for `DPNP_FN_ASTYPE_EXT` and `DPNP_FN_FLATTEN_EXT` or we can get rid of them?